File: SpotifyLocalStats.Server/Data/DTOs/AggregateDtos/AggregateTrackDto.cs
using SpotifyLocalStats.Server.Models;

namespace WebApi.Data.DTOs.NewFolder
{
    public class AggregateTrackDto : AggregateBaseDto
    {
        public List<TimeOfDayStatDto<AggregatedTrack>> TimeOfDayStats { get; set; }
    }
}

File: webapp/src/types/DTOs/AggregateDto/AggregateTrackDto.ts
import type { TimeOfDayStatDto } from "../TimeOfDayDtos";
import type { AggregateBaseDto } from "./AggregateBaseDto";

export interface AggregateTrackDto extends AggregateBaseDto {
  timeOfDayStats: TimeOfDayStatDto<AggregateTrackDto>[];
}
